packages/salesforcedx-vscode-core/src/commands/templates/libraryBaseTemplateCommand.ts

```diff
@@ -132,13 +131,21 @@ export abstract class LibraryBaseTemplateCommand<T>
     const templateService = TemplateService.getInstance(cwd);
     let customOrgMetadataTemplates;
 
-    const configValue = await ConfigUtil.getConfigValue(
-      OrgConfigProperties.ORG_CUSTOM_METADATA_TEMPLATES
+    const sfdxConfigValue = await ConfigUtil.getConfigValue(
+      'customOrgMetadataTemplates'
     );
-    if (configValue === undefined) {
+    // See W-11445939
+    // this key should be updated to use
+    // OrgConfigProperties.ORG_CUSTOM_METADATA_TEMPLATES
+    // once Templates library is updated and compatible like so:
+    // const sfConfigValue = await ConfigUtil.getSfConfigValue(
+    //   OrgConfigProperties.ORG_CUSTOM_METADATA_TEMPLATES
+    // );
+
+    if (sfdxConfigValue === undefined) {
       customOrgMetadataTemplates = undefined;
     } else {
-      customOrgMetadataTemplates = String(configValue);
+      customOrgMetadataTemplates = String(sfdxConfigValue);
```

Comment:
nit: this if/else could be`customOrgMetadataTemplates = sfdxConfigValue ? String(sfdxConfigValue) : undefined;`